src/igvfd/audit/file.py

```diff
@@ -112,3 +112,35 @@ def audit_file_no_file_format_specifications(value, system):
             f'has no `file_format_specifications`.'
         )
         yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
+
+
+@audit_checker('IndexFile', frame='object')
+@audit_checker('AlignmentFile', frame='object')
+@audit_checker('TabularFile', frame='object')
+@audit_checker('ReferenceFile', frame='object')
+@audit_checker('ModelFile', frame='object')
+@audit_checker('SequenceFile', frame='object')
+def audit_file_missing_anvil_url(value, system):
+    '''
+    [
+        {
+            "audit_description": "Released and archived files that are restricted access are expected to be loaded in anvil.",
+            "audit_category": "missing anvil url",
+            "audit_level": "INTERNAL_ACTION"
+        }
+    ]
+    '''
+    object_type = space_in_words(value['@type'][0]).capitalize()
+    audit_message = get_audit_message(audit_file_missing_anvil_url, index=0)
+    if value.get('status') in ['released', 'archived'] and value.get('controlled_access', False) and not value.get('anvil_url'):
```

Comment:
Do we want to put archived data into Anvil?  I had not thought about it, i was focused on released state.

Reply:
archived is functionally the same as released so I see no reason why it should be deposited to anvil (and therefore made accessible).

Reply:
should not be*

Reply:
My thought had been that archived means that it is no longer interesting so I would wonder why we would want to go to the trouble of propogating it to another site.    I had even wondered if we should remove archived data from the Anvil site as we have no way to indicate that it is different from released data